bin/yaml/android-java/fetch_art_release_from_head.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Fetches ART binaries and ART bootclasspath jars from AOSP head.
+# Saves the result at `art_release.zip`.
+
+set -e
+
+URL=https://ci.android.com/builds/latest/branches/aosp-main/targets/aosp_arm64-trunk_staging-userdebug/view/BUILD_INFO
```

Comment:
will this binary run on an x86 linux machine?

Reply:
Yes, the binary is built for x86_64. The arm64 here refers to the compiler output, not that the binary runs on an arm64 machine.